doc/appendices/administration.rst

```diff
@@ -0,0 +1,148 @@
+   is responsible for steering the technical implementation of the codebase to
+   meet the long-term project goals. One key responsibility of the code lead is
+   to set up milestones for releases and execute the release process described
+   below.
+
+Maintainer
+   Maintainers should be familiar with most or all parts of the codebase and
+   are responsible for merging pull requests and closing issues.
+
+Core team member
+   The "core team" are those currently funded to work on Celeritas. Core team
+   members are responsible for reviewing pull requests in accordance with the
+   :ref:`contributing guidelines <contributing>`. They should regularly
+   contribute new code or publish new results using Celeritas.
+
+Contributor
+   Anyone can submit a pull request that conforms to the contribution
+   guidelines! Contributors are recognized by including their handle and pull
+   requests in release notes but do not otherwise have a formal role.
+
+
+Role change process
+-------------------
+
+Adding or removing a member of the "core team" must be done by consensus of the
+leadership team (or if the core team member wants to remove themself). Adding
+maintainers can be done at the whim of the code lead. The `team list`_ on
+GitHub is the official record of roles.
+
+.. _team list: https://github.com/orgs/celeritas-project/teams
+
+Release process
+===============
+
+Celeritas uses `Semantic Versioning`_ to enumerate releases. During its initial
+development phase, ``0.x.0`` is a major release and ``0.x.z`` is a patch
+release. When Celeritas is declared stable, ``x.0.0`` is a major release,
+``x.y.0`` is a minor release, and ``x.y.z`` is a patch release.
+
+Major and minor releases (including 0.x.0 development releases) must have a
+milestone in the git issue tracker with a list of issues that can be assigned.
+Only major releases can deprecate or remove features and change
+:ref:`public-facing APIs <api>`. Both major and minor releases should include
+notable improvements to the code.
+
+Patch releases can be created at any time but should typically include at least
+one critical bug fix or several substantial fixes. Patch releases should focus
+almost exclusively on fixes and should generally not include new features or
+other major code changes.
+
+.. _Semantic Versioning: https://semver.org
+
+
+Releases can be created from the main branch (major, minor, patch) or a
+"backport" branch (minor, patch). The following process must be followed (and
+may need iteration to converge) for each release.
+
+- Ensure all CI jobs passed for the release in question. This is automatic for
+  releases from the main branch (since every pull request must pass) but should
+  be checked manually for backports.
+- Run regression tests on Summit (for performance testing), Crusher (for HIP
+  testing), and an additional machine with debug assertions enabled (e.g.
+  Wildstyle). Postpone the release
```

Comment:
Do we want to have a separate, easily editable list of machine names here so we don't have to continually go back to this core section of documentation to update `Summit`, `Crusher`, etc?  (I think we had tons of references to Jaguar in our original developer manual in Shift, they are probably still there!).

Reply:
This checklist is something that should be referenced at every release (it's a checklist I want to go through to make sure I'm doing everything) and can be updated as necessary. One thing I could do is to turn this section into a github issue template, so we'd have an issue with a checklist for every major/minor/patch release...